addons/website/tests/test_website_favicon.py

```diff
@@ -34,3 +37,16 @@ def test_01_website_favicon(self):
         self.assertEqual(image.format, 'ICO')
         self.assertEqual(image.size, (256, 256))
         self.assertEqual(image.getpixel((0, 0)), bg_color)
+
+
+@tagged('-at_install', 'post_install')
+class TestWebsiteHttp(HttpCase):
+    def test_website_default_social_media(self):
```

Comment:
Shouldn't the test also try with a `res_company` social media field reset to `None` before the new website creation ?

Reply:
It actually should try with this case only, as the company social media fields are actually set only with demo data.
Somehow the enterprise build is failing because of that, it looks like the community build is done against a company which doesn't have social set while in enterprise the company has those fields set.

I made it explicit so demo date don't get in the way of the test.